src/Symfony/Component/Message/Transport/Serialization/DecoderInterface.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Message\Transport\Serialization;
+
+/**
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ */
+interface DecoderInterface
+{
+    /**
+     * Decode the message from an encoded-form. The `$encodedMessage` parameter is a key-value array that
```

Comment:
`Decodes`

I would move the second sentence in the description (and remove the ```).

Reply:
Not sure what you mean about "remove the ```".